tests/TestCase/Database/Type/DateTimeTypeTest.php

```diff
@@ -94,6 +94,16 @@ public function testToPHPString()
         $this->assertEquals('12', $result->format('H'));
         $this->assertEquals('13', $result->format('i'));
         $this->assertEquals('14', $result->format('s'));
+
+        $this->type->setTimezone('Asia/Kolkata'); // UTC+5:30
```

Comment:
Why is the resulting datetime' hour 6 if the offset is 5:30?

Reply:
Since this timezone it UTC+5:30 you need to go back 5:30 hours to get UTC time 

Going back 5:30 hours from 12:00 gives you 6:30.

Reply:
Derp. My bad.

Reply:
Should we have a similar test for the `manyToPhp()` method?

Reply:
Done